Jenkinsfile

```
@@ -1,6 +1,6 @@
 // Build the plugin using https://github.com/jenkins-infra/pipeline-library
 buildPlugin(useContainerAgent: true, configurations: [
-        [platform: 'linux', jdk: '8'],
+        [platform: 'linux', jdk: '21'],
```

Need to remove the `2.342` from the configuration because the minimum Jenkins version is greater than 2.342.
```suggestion
        [platform: 'linux', jdk: 21],
```

The test that I ran on ci.jenkins.io used this Jenkinsfile:

```
// Build the plugin using https://github.com/jenkins-infra/pipeline-library
buildPlugin(useContainerAgent: true, configurations: [
        [platform: 'linux', jdk: 21],
        [platform: 'linux', jdk: 11],
        [platform: 'linux', jdk: 17],
])
```